L1Trigger/L1THGCal/python/hgcalVFEProducer_cfi.py

```diff
@@ -46,8 +48,8 @@
                      scintillatorCellLSB_MIP = cms.double(float(adcSaturationBH_MIP.value())/(2**float(adcNbitsBH.value()))),
                      # cell thresholds before TC sums
                      # Cut at 3sigma of the noise
-                     thresholdsSilicon = cms.vdouble([3.*x for x in digiparam.HGCAL_noise_fC.values.value()]),
-                     thresholdScintillator = cms.double(3.*digiparam.HGCAL_noise_MIP.value.value()),
+                     thresholdsSilicon = cms.vdouble([sensor_thresh*x for x in digiparam.HGCAL_noise_fC.values.value()]),
```

Comment:
Changes to be reverted to avoid future conflicts: here are changes happening here in parallel.